packages/telestion-client-common/package.json

```diff
@@ -54,9 +54,11 @@
 		}
 	],
 	"dependencies": {
+		"@react-spectrum/table": "^3.0.0-alpha.12",
```

Comment:
As `@adobe/react-spectrum` is a peer dependency, could we please make `@react-spectrum/table` a peer dependency as well, please?

Reply:
Done in [`7627af2` (#645)](https://github.com/wuespace/telestion-client/pull/645/commits/7627af2391efa23be3f546192eca17c3a4b6b045).
I also added `@react-spectrum/table` to the `@wuespace/telestion-client-template` package.